polkadot/xcm/pallet-xcm-benchmarks/src/generic/mod.rs

```diff
@@ -97,6 +98,14 @@ pub mod pallet {
 		/// If set to `Err`, benchmarks which rely on a universal alias will be skipped.
 		fn alias_origin() -> Result<(Location, Location), BenchmarkError>;
 
+		/// An XCM used to exercise the barrier logic in benchmarks.
```

Comment:
More information on how to set this would be helpful for people having to implement it.

From what I gather, it's an XCM that doesn't pass the `Barrier` and it also weighs a lot.